templates/artist/artist_detail.html

```diff
@@ -171,11 +184,18 @@ <h1 class="header-font">{{ artist.name }}</h1>
             <hr>
         </div>
         <!-- Current Investors -->
-        <h2>Current Investors</h2>
-        <ul>
+        <ul class="vertical menu" data-accordion-menu>
+          <li>
+        <a href= "#"><h2>CURRENT INVESTORS</h2></a>
+            <ul class="menu vertical nested is-active">
             {% for investor in investors %}
-                <li>{% if investor.public_profile_url %}<a href="{{ investor.public_profile_url }}"><img src="{{ investor.avatar_url }}" alt="{{ investor.name }}" /> {{ investor.name }}</a>{% else %}{{ investor.name }}{% endif %}: ${{ investor.total_investment }}</li>
+                <li>{% if investor.public_profile_url %}
```

Comment:
Cool. That makes sense. The only thing I noticed is when removing the <p> tab it breaks up the amount invested onto two lines. Is there a way to get the amount invested on the same line? Also, do you like the avatar on the side more than on top? I'm cool with either way